src/medals/main.cljs

```diff
@@ -0,0 +1,39 @@
+(ns medals.main
```

Comment:
Are you using `leiningen`? I don't see any `project.clj` file. If no please migrate this to be `leiningen` project. We are using `leiningen`, so it will help you get more familiar with this.

https://leiningen.org/

But if you are using `leiningen` please include `project.clj` file in repository.